external/model-preparation-algorithm/tests/ote_cli/test_small_object_inst_seg.py

```diff
@@ -0,0 +1,151 @@
+"""Tests for MPA Class-Incremental Learning for instance segmentation with OTE CLI"""
```

Comment:
The name 'test_small_*' doesn't come to mind at once. Any other names to replace?

Reply:
test_tiling_inst_seg?

Reply:
I think it looks better.